js/tests/unit/button.js

```diff
@@ -110,6 +110,18 @@ $(function () {
     assert.ok($btn.hasClass('active'), 'btn has class active')
   })
 
+  QUnit.test('should prevent toggle active when disabled btn children are clicked', function (assert) {
```

Comment:
Isn't the "children" part unnecessary? It happens when clicking the `<button>` itself too, right?